arches/app/templates/base.htm

```diff
@@ -44,8 +44,9 @@
     <link rel="apple-touch-icon" sizes="120x120" href="{% webpack_static 'favicons/apple-touch-icon-120x120.png' %}" />
     <link rel="apple-touch-icon" sizes="152x152" href="{% webpack_static 'favicons/apple-touch-icon-152x152.png' %}" />
     <link rel="apple-touch-icon" sizes="180x180" href="{% webpack_static 'favicons/apple-touch-icon-180x180.png' %}" />
-
```

Comment:
nit: unnecessary line deletion

Sorry to be picky about this but it's a part of core that will likely not be touched again for a while. We really should have a hook that removes unnecessary whitespace so this is less of an issue